chall04/mlindhol.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# python 3.7.4 Windows 10
+
+import sys
+
+
+def input_error():
+    print("Bad input, n > 0 and only '.' ' ' and '#' allowed")
+    exit(-1)
```

Comment:
I would suggest avoiding creating a function for that and use sys.exit() with your message as argument of sys.exit(); 
https://docs.python.org/2/library/sys.html#sys.exit